spec/facades/movie_facade_spec.rb

```diff
@@ -0,0 +1,51 @@
+require "rails_helper"
+
+describe "Movie Facade" do
+  it "returns a list of searched movies objects" do
+    VCR.use_cassette('movies_by_search', :record => :new_episodes) do
```

Comment:
Just for posterity... that `:record => :new_episodes` addition is so the VCR records new information when it's run again